deployments/dev/images/secondary/environment.yml

```diff
@@ -7,69 +7,69 @@ channels:
 dependencies:
 - python==3.11.*
 - git==2.39.1
+- jupyterhub==5.0.0
 - jupyter-resource-usage==1.0.0
 - jupyterlab==4.0.11
 - jupyterlab-favorites==3.0.0
+# Install myst for generating dashboards
+- jupyterlab-myst==2.4.2
 - jupyterlab_server==2.23.0
 - jupyterlab_widgets==3.0.8
 - jupyter_server==2.7.0
 - nbgitpuller==1.2.1
 - notebook==7.0.7
 - folium==0.14.0
-- h5netcdf==1.0.2
+  #- h5netcdf==1.0.2
 - ipywidgets==8.0.7
-- jupysql==0.8.0
+  #- jupysql==0.8.0
 - jupyter-archive==3.4.0
 - matplotlib==3.7.1
-- mdit-py-plugins==0.4.0
+# Install myst markdown for generating dashboards
+- mystmd==1.2.5
+  #- mdit-py-plugins==0.4.0
 - numpy==1.24.2
 - pandas==2.0.2
 - plotly==5.13.1
 - requests==2.28.2
 - scikit-image==0.19.3
-- scikit-learn==1.2.2
-- scipy==1.10.1
+  #- scikit-learn==1.2.2
+  #- scipy==1.10.1
 - seaborn==0.12.2
-- statsmodels==0.14.0
-- sqlalchemy==2.0.16
-- mlxtend==0.23.0
+  #- statsmodels==0.14.0
+  #- tensorflow-cpu==2.12.1
+  #- sqlalchemy==2.0.16
+  #- mlxtend==0.23.0
 # Spring 2024 data 100
-- pytorch==2.1.2
-- cpuonly==2.0
-- transformers==4.37.1
+#- pytorch==2.1.2
+#- cpuonly==2.0
+#- transformers==4.37.1
 # Spring 2024 table demos
-- lxml==5.1.0
+#- lxml==5.1.0
 # Spring 2024 Econ 148 Packages
 - geopandas==0.14.2
 - geopy==2.4.1
-- lifelines==0.27.8
+  #- lifelines==0.27.8
 - pycountry==22.3.5
-# Install voila for generating dashboards
+# Install voila and voici for generating voila dashboards
 - voila==0.5.7
-# Install myst for generating dashboards
-- mystmd==1.2.5
-- jupyterlab-myst==2.4.2
-#- ipywidgets==8.1.3
-#- pytest-notebook==0.10.0
-#- gh-scoped-creds==4.1
-#- ydata-profiling==4.8.3
-#- otter-grader==5.5.0
-#- python-duckdb==1.0.0
-#- jupyterhub==5.0.0
-- quarto==1.4.550
-- pip
-- pip:
+- voici==0.6.1
```

Comment:
voici is available as a conda package so should not be under pip.

Reply:
Actually, voici is part of conda block. I am not sure why the Github snapshot shows it is under pip